src/memusage.h

```diff
@@ -25,6 +25,64 @@ namespace memusage
 /** Compute the total memory used by allocating alloc bytes. */
 static size_t MallocUsage(size_t alloc);
 
+/** Wrapping allocator that accounts accurately.
+ *
+ * To use accounting, the AccountAllocator(size_t&) constructor must be used.
+ * Any containers using such an allocator, and containers moved constructed or
+ * assigned from such containers will then increment/decrement size_t when
```

Comment:
65fc229fabbc7889baca9c50ee35507543a6682e
nit:
```suggestion
 * Any containers using such an allocator, and containers move constructed or
 * move assigned from such containers will then increment/decrement size_t when
```